src/autopas/selectors/tuningStrategy/TuningStrategyLoggerWrapper.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file TuningStrategyLoggerWrapper.cpp
+ * @author humig
+ * @date 24.09.2021
+ */
+
+#include "TuningStrategyLoggerWrapper.h"
+
+#include "autopas/selectors/tuningStrategy/TuningLogEntry.h"
+#include "autopas/utils/Timer.h"
+
+namespace autopas {
+
+TuningStrategyLoggerWrapper::TuningStrategyLoggerWrapper(std::unique_ptr<TuningStrategyInterface> actualTuningStrategy,
+                                                         const std::string &outputSuffix)
+    : _actualTuningStrategy(std::move(actualTuningStrategy)) {
+  std::stringstream filename;
+  filename << "tuningLog-";
+  filename << utils::Timer::getDateStamp();
+  filename << '-' << outputSuffix;
+  filename << ".txt";
+  _logOut.open(filename.str());
+}
+
+TuningStrategyLoggerWrapper::~TuningStrategyLoggerWrapper() { _logOut.flush(); }
```

Comment:
this should probably also close the file?

Reply:
Should happen automatically in the destructor of std::ofstream ("destructs the basic_ofstream and the associated buffer, closes the file" from https://en.cppreference.com/w/cpp/io/basic_ofstream).